pkg/api/mlflow/dao/repositories/helpers_test.go

```diff
@@ -53,3 +53,53 @@ func Test_makeParamConflictPlaceholdersAndValues(t *testing.T) {
 		assert.Equal(t, tt.expectedValues, values)
 	}
 }
+
+func TestBuildJsonCondition(t *testing.T) {
+	tests := []struct {
+		name             string
+		dialector        string
+		jsonColumnName   string
+		jsonPathValueMap map[string]string
+		expectedSQL      string
+		expectedArgs     []interface{}
+	}{
+		{
+			name:           "Postgres",
+			dialector:      "postgres",
```

Comment:
we have a `SQLiteDialectorName` and `PostgresDialectorName` in `database/constants.go`, can you please re use them?

Reply:
ok